### PR TITLE
Bugfix/header-updates

### PR DESCRIPTION
**Pull request by lezegom**
#### How to test it?

[Workspace]()

### Screenshots/Video example usage:

- Desktop

- Tablet

- Movil

### CODE REVIEW

File: node/utils/Hub.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default class RequestHub extends ExternalClient {
       headers: {
         Accept: 'application/json',
         'Content-type': 'application/json',
-        Authorization: context.adminUserAuthToken ?? context.authToken ?? '',
```
Comment (Gift-retyu) [inline node/utils/Hub.ts:36]: why removing this?

Comment (lezegom) [reply node/utils/Hub.ts:36]: Sego's update to master has a different `Authorization` [here](https://github.com/TFG-Labs/store-locator/pull/16/files#diff-186b72cb363c858aaadd387ab717fb477edf665874b8273ed2224805357c61d7)

**Comment by SegopotsoMakhutja** [comment]
what's the workspace to test?

**Comment by lezegom** [comment]
> what's the workspace to test?

@SegopotsoMakhutja 
WS: https://storefinder--thefoschini.myvtex.com/store-finder

**Pull request closed without merging**